SampleApp/ConsistencyManagerDemo-tvOSUITests/ConsistencyManagerDemo_tvOSUITests.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ConsistencyManagerDemo_tvOSUITests.swift
+//  ConsistencyManagerDemo-tvOSUITests
+//
+//  Created by Andrew Sowers on 1/6/17.
+//  Copyright © 2017 LinkedIn. All rights reserved.
+//
+
+import XCTest
+
+class ConsistencyManagerDemo_tvOSUITests: XCTestCase {
```

Comment:
Can we remove these UI tests? I highly doubt we'll ever write UI tests for the demo app and it's just cruft.